test/e2e/_data/main/rebuild_canned_blocks_main.go

```diff
@@ -24,9 +24,13 @@ func main() {
 
 	clearBlocksFile()
 
-	n := e2e.NewInProcessE2EAppNetwork(42, e2e.NewLoggerRandomer())
+	n := e2e.NewInProcessE2EAppNetwork(42, e2e.NewLoggerRandomer(), "/tmp/dummy")
```

Comment:
Is this going to write on `/tmp/dummy`? If so, I think it should use a more specific name.
Also, can you elaborate a bit on why this was suddenly necessary?

Reply:
replacing with `""`, since they are similar in this context.

Reply:
This parameter was introduced by @netoneko while working on javascript processor. In this PR I needed to replace the `blocks` file so I did the minimal changes to make this `main` compile